CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -36,6 +36,8 @@ @implementation CPAppBannerModuleInstance
 
 NSInteger currentScreenIndex = 0;
 
+NSString *currentEventId = @"";
```

Comment:
Can you explain why we need to store the currentEventId here? :)

Reply:
`currentEventID` had been taken to handle which track event needs to be called and that a particular banner will be displayed.

Example:-

I had taken two buttons.

- On the first button, I am calling `trackevent ("iOS_first_track_event")` and that particular banner was displayed, and it is correct.
- On the second button, I am calling the second track event `trackevent ("iOS_second_track_event")`, and the thing was that the first previous track event banner was displaying, and after that, the second track event's banner was displaying.

So to display the right banner using the particular track event ID, we need to keep this variable.